Tests/SwiftProtobufTests/Test_BinaryDecodingOptions.swift

```diff
@@ -199,14 +199,14 @@ final class Test_BinaryDecodingOptions: XCTestCase {
         //   }
         do {
             let msg1 = try SwiftProtoTesting_TestAllExtensions(
-              serializedBytes: inputSubMessage,
+                serializedBytes: inputSubMessage,
```

Comment:
looks like some extra space in a few on the final diffs

Reply:
Some of the files have become messed up at some point: there are files exclusively with 2 spaces, some with exclusively 4, and some with mixed 2 and 4. This adds a lot of churn when using refactoring tools in Xcode (such as renaming funcs). At some point we should make sure the spacing across the whole repo is consistent, and/or consider introducing something such as swift-format.